### DISCUSSION TITLE
cuy university semester 4 ||

### DISCUSSION THREAD

**Discussion by tventven** [deaafrizal/forum#18] [Troubleshooting Room]
izin bertanya bang,saya baru ngikutin cuy university semerter 4 sesi 1 bang.
Waktu saya mau nyoba "npm run dev" malah error bang. sempet muncul "cuy anime list" sebentar, setelah itu error
<img width="959" alt="Screenshot 2023-11-30 083132" src="https://github.com/deaaprizal/forum/assets/121137257/e362a5ef-c725-4aec-803f-61fee3062e08">
<img width="930" alt="Screenshot 2023-11-30 083545" src="https://github.com/deaaprizal/forum/assets/121137257/35af4512-89c7-4c07-9c41-e11a7e40b74a">

**Comment by deaafrizal** [comment]
biar temen2 yang lain bisa bantu juga, boleh di share dulu environment nya: node js versi berapa, apa udah install node module nya, tahapan jalanin projeknya kaya gimana, apakah install dari 0 atau dari repository dan lain2 nya.

**Reply by tventven** [reply]
Untuk nodejs nya saya pkae versi 20.10.0, dan projeknya saya install dari 0

**Reply by deaafrizal** [reply]
coba pake nod ejs versi 18 bro, kalau gak mau uninstall bisa pake yang namanya nvm untuk switch versi node nya.

**Reply by tventven** [reply]
Terimakasih bang,nanti saya coba switch ke versi 18

**Reply by deaafrizal** [reply]
siap berkabar aja disini kalau berhasil biar pada tau juga cara solve nya 👌
